src/test/java/org/omnifaces/test/cdi/param/ParamIT.java

```diff
@@ -73,6 +73,7 @@ public class ParamIT extends OmniFacesIT {
     public static WebArchive createDeployment() {
         return buildWebArchive(ParamIT.class)
             .withWebXml(WebXml.withInterpretEmptyStringSubmittedValuesAsNull)
+            //.withFacesConfig(FacesConfig.basic)
```

Comment:
Why?

Reply:
I don't remember at the moment, but months ago 
I had trouble running this test....

I'll check this again

Reply:
Please don't touch existing tests as long as they all pass in github actions and the code behind is not modified.